src/writer/direntPool.h

```diff
@@ -46,11 +46,22 @@ namespace zim
           }
         }
 
-        Dirent* getDirent() {
+        Dirent* getClassicDirent(char ns, const std::string& path, const std::string& title, uint16_t mimetype) {
```

Comment:
My comment applies to the `DirentPool` class, but github doesn't allow commenting on code that is more than 3 lines away from changes.  If you don't make `DirentPool` non-copyable it may explode sooner or later.

Reply:
Done